docs/features/shorthand.md

```diff
@@ -0,0 +1,326 @@
+
+:::plot
+```js
+Plot.barY(numbers).plot()
+```
+:::
+
+Like Plot.barY, [Plot.cellX](../marks/cell.md) implies that *x* is ordinal. But now instead of a *y* channel the numeric value is encoded as the *fill* color. The default quantitative color scheme is *turbo*; higher values are reddish, and lower values blueish.
+
+:::plot
+```js
+Plot.cellX(numbers).plot()
+```
+:::
+
+If we don’t care about the order of our data and we instead just want to look at the one-dimensional distribution of values, we can use [Plot.dotX](../marks/dot.md).
+
+:::plot
+```js
+Plot.dotX(numbers).plot()
+```
+:::
+
+Alternatively, we can use [Plot.ruleX](../marks/rule.md) to draw a vertical rule at each value. In this case, Plot.ruleX behaves identically to [Plot.tickX](../marks/tick.md). (If there *were* a *y* channel, then Plot.tickX would imply that *y* is ordinal whereas Plot.ruleX would imply that *y* is quantitative.) It is common to use the rule shorthand to annotate special *x* or *y* values in plots, such as *y* = 0, in conjunction with other marks.
+
+:::plot
+```js
+Plot.ruleX(numbers).plot()
+```
+:::
+
+:::plot
+```js
+Plot.tickX(numbers).plot()
+```
+:::
+
+We could even use [Plot.vectorX](../marks/vector.md) here to draw little up-pointing arrows. (Typically the vector mark is used in conjunction with the *rotate* and *length* options to control the direction and magnitude of each vector.)
+
+:::plot
+```js
+Plot.vectorX(numbers).plot()
+```
+:::
+
+While not particularly readable due to occlusion, we can use [Plot.textX](../marks/text.md) to draw a label at each value, too.
+
+:::plot
+```js
+Plot.textX(numbers).plot()
+```
+:::
+
+For a more formal method of summarizing a one-dimensional distribution, we can use [Plot.boxX](../marks/box.md) to create a horizontal boxplot. The gray band represents the interquartile range; the black whiskers show the extrema (not including outliers); and the thick black stroke represents the median; any outliers (none in this dataset) are drawn as dots.
+
+:::plot
+```js
+Plot.boxX(numbers).plot()
+```
+:::
+
+Some of Plot’s transforms support shorthand syntax, too. For example, we can use Plot.rectY with [Plot.binX](../transforms/bin.md) to generate a histogram—another common way to visualize a one-dimensional distribution.
+
+:::plot
+```js
+Plot.rectY(numbers, Plot.binX()).plot()
+```
+:::
+
+Similarly [Plot.groupX](../transforms/group.md) can be used to group and count ordinal data, such as the frequency of bases in a random DNA sequence.
+
+```js
+gene = "AAAAGAGTGAAGATGCTGGAGACGAGTGAAGCATTCACTTTAGGGAAAGCGAGGCAAGAGCGTTTCAGAAGACGAAACCTGGTAGGTGCACTCACCACAG"
+```
+
+:::plot
+```js
+Plot.barY(gene, Plot.groupX()).plot()
+```
+:::
+
+And here’s the [dodge transform](../transforms/dodge.md) for a beeswarm plot:
+
+:::plot
+```js
+Plot.dotX(numbers, Plot.dodgeY()).plot()
+```
+:::
+
```

Comment:
```suggestion

## Tuples

```

(maybe with a "Lists" subheader at the top?)

Reply:
Added “One dimension”, “Two dimensions”, and “Caveats” as sections.